tools/generator/cmd/v2/release/releaseCmd.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package release
+
+import (
+	"fmt"
+	"log"
+	"path/filepath"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/tools/generator/cmd/v2/common"
+	"github.com/Azure/azure-sdk-for-go/tools/generator/flags"
+	"github.com/Azure/azure-sdk-for-go/tools/generator/repo"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// Release command
+func Command() *cobra.Command {
+	releaseCmd := &cobra.Command{
+		Use:   "release-v2 <azure-sdk-for-go directory> <azure-rest-api-specs directory> <rp-name> <namespaceName>",
```

Comment:
```suggestion
		Use:   "release-v2 <azure-sdk-for-go directory> <azure-rest-api-specs directory> <rp-name> [namespaceName]",
```
since namespaceName is optional